end-to-end-tests/cypress.config.ts

```diff
@@ -3,20 +3,39 @@ const { generateZapReport } = require('./cypress/plugins/generateZapReport')
 const { verifyDownloadTasks } = require('cy-verify-downloads')
 
 export default defineConfig({
+
+  reporter: 'cypress-multi-reporters',
+  reporterOptions: {
+    reporterEnabled: 'mochawesome',
+    mochawesomeReporterOptions: {
+      reportDir: 'cypress/reports/mocha',
+      quite: true,
+      overwrite: false,
```

Comment:
Overwrite being false, does this mean there will be a collection of these growing overtime, does that incur a ops burden to clear it every now and then?

Reply:
As we are running the cypress test from GitHub the files are not saved onto the disk